pages/[network]/[address]/review.tsx

```diff
@@ -152,23 +156,44 @@ export async function getStaticPaths() {
 export const getStaticProps = async (
   context: GetStaticPropsContext<QueryParams>,
 ) => {
+  let network = context.params?.network;
   const address = context.params?.address;
-  const chainId = context.params?.chainId;
 
   let questStatuses: graphql.QuestStatusInfoFragment[] = [];
   let questChain = null;
-  if (chainId && address) {
-    try {
-      questStatuses = address
-        ? await getStatusesForChain(chainId, address)
-        : [];
-      questChain = address ? await getQuestChainInfo(chainId, address) : null;
-    } catch (error) {
-      // eslint-disable-next-line no-console
-      console.error(
-        `Could not fetch quest chains/statuses for address ${address}`,
-        error,
-      );
+
+  if (network && CHAIN_URL_MAPPINGS[network]) {
+    network = CHAIN_URL_MAPPINGS[network];
```

Comment:
shouldn't we return null or fail if `CHAIN_URL_MAPPINGS[network]` is not found? 
or we should find a way to redirect to the new networkName url when the user is inputting the hexstring chainId

Reply:
redirecting sounds like a good idea, yeah. but i mainly put this code there to safeguard urls that will have the network ids instead of names, like this: `https://questchains.xyz/chain/0x89/0x0bf17bedbd42f6cf89f4064d5d70da1e44f70f14`